frontend/front/src/api/apiSlice.js

```diff
@@ -311,6 +322,22 @@ export const apiSlice = createApi({
         ...result.map(({ id }) => ({ type: "PropertyAssessment", id })),
       ],
     }),
+    //sessions section of the slice
+    loginUser: builder.mutation({
+      query: ({ email, password }) => ({
+        url: "/users/sign_in",
+        method: "POST",
+        // TODO: should this be hashed??
+        body: { user: { email, password } },
```

Comment:
Do we need to like hash passwords in transit? Or is HTTPS sufficient?

Reply:
This **shouldn't** be hashed – we need the plaintext password, and rely on the HTTPS/TLS connection to get it safe in transit.